orocos.osdeps

```diff
@@ -93,7 +93,10 @@ graphviz:
     macos-brew: graphviz
 
 gccxml:
-    debian,ubuntu: gccxml
+    debian: gccxml
```

Comment:
> it must be ignore, when it is nonexistent, the dependency chain fails because it is required but nonexistent

Which dependency chain ? Nothing should not depend on gccxml if we're not using it.